docs/sources/example-rules/dropbox-business/dropbox-business.yaml

```diff
@@ -47,6 +49,8 @@ endpoints:
           - "$.entries[*]['name', 'path_lower', 'path_display', 'preview_url', 'content_hash']"
           - "$.entries[*].export_info['export_as']"
           - "$.entries[*].file_lock_info['lockholder_name']"
+    supportedHeaders:
+      - "Dropbox-API-Select-User"
```

Comment:
if this is common case, I'm OK to have this as a top-level property as alternative? (eg, on `Rules`, rather than `Endpoint`; and for each endpoint we allow super-set of headers specified on `Rules` + `Endpoint`?)

Reply:
Good point, let's see when I revisit dropbox; ATM for Salesforce is for endpoints related with SOQL.